zzzz_modular_occulus/code/game/objects/items/weapons/traitor_weapons.dm

```diff
@@ -0,0 +1,231 @@
+/obj/item/twohanded/garrote // 12TC traitor item
+	name = "fiber wire"
+	desc = "A length of razor-thin wire with an elegant wooden handle on either end.<br>You suspect you'd have to be behind the target to use this weapon effectively."
+	icon = 'zzzz_Modular_occulus/icons/obj/weapons.dmi'
+	icon_state = "garrot_wrap"
+	w_class = ITEM_SIZE_TINY
+	var/mob/living/carbon/human/strangling
+	var/improvised = 0
+	var/garrote_time
+
+
+/mob/living/proc/grabbedby(mob/living/carbon/user, supress_message = FALSE)
```

Comment:
There's no reason to define this on the `/mob/living` level, as the stranglee can only be a human, as it is written now.

Reply:
uh... You can strangle roachs. Which aren't human.

Reply:
L88

Reply:
You can also strangle monkeys.

Reply:
Then put it under `/mob/living/carbon/human`. Monkeys are a subtype of human

Reply:
In this case putting it under /mob/living/carbon/human will cause this code to only be checked if you are, in fact, strangling a human.  There isn't much reason not to do it unless you want to make it possible to garrote roaches later.

Reply:
Adjusted